Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Entries/RootAssetBrowserEntry.cpp

```diff
@@ -5,6 +5,7 @@
  * SPDX-License-Identifier: Apache-2.0 OR MIT
  *
  */
+#include "AzCore/IO/Path/Path_fwd.h"
```

Comment:
This should angle brackets for the include
```suggestion
#include <AzCore/IO/Path/Path_fwd.h>
```

Reply:
Fixed. It actually doesn't need `Path_fwd.h` at all, it includes `Path.h` already.